aries_vcx/src/handlers/trust_ping/mod.rs

```diff
@@ -0,0 +1,112 @@
+
+
+use crate::error::{VcxError, VcxErrorKind, VcxResult};
+
+use crate::messages::a2a::MessageId;
+use crate::messages::trust_ping::ping::Ping;
+use crate::messages::trust_ping::ping_response::PingResponse;
+use crate::protocols::SendClosure;
+use crate::utils::uuid;
+
+pub mod util;
```

Comment:
I find it nicer to put `mod`s first

Reply:
we generally use `pub mod` after `use` declarations across the codebase. But let's leave this up to `cargo fmt` to deal with